packages/suite/src/components/wallet/Pagination.tsx

```diff
@@ -53,6 +53,7 @@ interface PaginationProps {
     hasPages?: boolean;
     perPage: number;
     totalItems: number;
+    limitedPageList?: boolean;
```

Comment:
What about `isPageListLimited`? Sounds more like boolean

Reply:
I noticed there are similar naming on multiple places across PR, we don't have any strong rule how name these booleans, but with this hungarian notation (is, has,...) it's maybe easier to understand. Feel free to ignore it if you disagree.

Reply:
TL;DR: okay, will change it.

Long version: using verb `isPageListLimited` suggests a metod or a function call "you do something". Whereas in declarative programming style here, we "describe the state" so we "tell the react how things are" we don't ask ,we don't "ask it to act".

So:
```tsx
<MyComponent visible={false} /> 
```

in MyComponent
```tsx
function MyComponent() {
  function isVisible() {
    // do some crazy stuff here
    return false
  }
  
  
  const visible = isVisible()
  
  return (<SomeComponent parentVisible={visible} /> )
}
 
```

Yes, we litearlly had such conversation back in avocode.

Reply:
OK, this is also valid, let's discuss it later 👍

Reply:
wouldn't bother :D